### DISCUSSION TITLE
Examining of monsters giving the offensive and defensive stats available from their wikipage.

### DISCUSSION THREAD

**Discussion by greenpk2** [runelite/runelite#15594] [Ideas]
We have the examining of items giving the GE and High alch price for items. Can we have same useful information from monsters, where we get the stats listed? Would be nice when you just fighting some monsters and are curious. Ofcourse you can check wiki, but its a QOL update which already are available on items, why not monsters?

**Comment by Amascut** [comment]
i agree, would be pretty helpful for deciding attack styles against certain enemies depending on their defences

**Comment by lennerthoh** [comment]
Hi, 

There is a solution for this. 

If you add the plugin "DPS Calculator" (made by LlemonDuck), go to settings and turn on "Right-Click Entry".
If you now Right click "dps" on a monster the monster stats will popup.
